lib/FlashMem/FlashMem.h

```diff
@@ -140,6 +146,42 @@ namespace Flash
      */
     bool saveString(const uint8_t &address, const uint8_t &maxLength, const String &input);
 
+    /**
+     *  Retrieves an Unsigned Integer from the EEPROM.
```

Comment:
But only a 8-bit value!

Reply:
Makes sense to leave it as an 8-bit value or should it be expanded for a 32-bit value instead?